dpti/ti.py

```
@@ -828,9 +792,7 @@ def refine_task(from_task, to_task, err):
 
     from_ti = os.path.join(from_task, "ti.out")
     if not os.path.isfile(from_ti):
-        raise RuntimeError(
-            f"cannot find file {from_ti}, task should be computed befor refined"
-        )
+        raise RuntimeError(f"cannot find file {from_ti}, task should be computed befor refined")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)